_docs/using-beaker/index.md

```diff
@@ -1,28 +1,224 @@
 ---
-title: The peer-to-peer Web
+title: Welcome to Beaker
 section: usingBeaker
 sectionTitle: Using Beaker
 order: 1
+class: dark
 ---
 
-<img data-src="/img/docs/tour-intro.png" class="centered">
+Beaker is a new kind of browser that gives you the power to create websites,
+share files, and control your data. It's an evolving reimagination of the Web,
+and we're excited to have you on the journey with us. Welcome!
 
-Peer-to-peer websites are just like any other website. But instead of being served from a datacenter, files are hosted by peers on your network. This makes it easy for anybody to host a site.
+<img data-src="/img/logo/256x256.png" style="display: block; margin: 3em auto">
+
+<section markdown="1">
+
+## A peer-to-peer Web
+
+At the heart of Beaker is a **peer-to-peer network** for sharing all kinds of
+content.
+
+It lets you skip the middle man and share data directly with your
+friends and colleagues. P2P is fantastic for sharing information privately, and
+it works great for publishing to a broad audience too!
 
 <figure>
 <img data-src="/img/docs/tour-site-peer-count.jpg">
-<figcaption>Browsing to a peer-to-peer site shows the number of peers</figcaption>
+<figcaption>Peer-to-peer networking skips the middle man and shares data directly.</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## Using Beaker
+
+### Share personal files
+
+Create a site, add the files, and send the URL using a chat program. It's
+encrypted from end-to-end so that you have total privacy.
+
+### Make webpages
+
+Create a site, fill it with your
+HTML/CSS/JS, and publish the URL (eg on Twitter). Your users can save the sites
+permanently, so that they work offline. [You can host in the
+cloud too.](/docs/tutorials/host-outside-of-beaker.html)
+
+### Hack on P2P apps
+
+Create a site, then build your app
+using the [new Web APIs in Beaker](/docs/apis/). Every app is hostless, and
+ships with forkable source code.
+
+### Browse the Web
+
+Of course, Beaker works great as a regular Web browser too. It's built using
+Chromium, the Free and Open Source engine from Google Chrome.
+
+</section>
+
+<section markdown="1">
+
+## How to create a site
+
+If you want to share files, make webpages, or hack on apps, you'll need to
+create a site.
+
+<figure>
+<img data-src="/img/docs/tour-new-site.jpg">
+<figcaption class="thick">Click "New site" in the browser dropdown menu</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## Adding files to a site
+
+You can add files to a site by opening the folder in your explorer, then
+copying the files into the directory.
+
+<figure>
+  <img class="screenshot" src="/img/docs/tour-new-site-publish1.png"/>
+  <figcaption class="thick">Click "Open folder"</figcaption>
+</figure>
+
+</section>
+
+<section markdown="1">
+
+## Using the "Import files" button
```

Comment:
So I agree that these do look nice as individual cards, but this one in particular seems out of place as a standalone item. Maybe it should be included in the adding files to a site section.